test/unit/spec/narrative_core/jobCommChannel-spec.js

```diff
@@ -87,15 +47,14 @@ define(['jobCommChannel', 'base/js/namespace', 'common/runtime'], (
     }
 
     describe('The jobCommChannel widget', () => {
-        let runtime;
-
         beforeEach(() => {
-            runtime = Runtime.make();
+            this.bus = Runtime.make().bus();
```

Comment:
use `beforeEach(function() {...})` so that the fat arrow's `this` context doesn't usurp that provided by Jasmine